contracts/sol6/SanityRatesGasPrice.sol

```diff
@@ -0,0 +1,99 @@
+pragma solidity 0.6.6;
+
+import "./IKyberSanity.sol";
+import "./utils/Utils5.sol";
+import "./utils/Withdrawable3.sol";
+
+/**
+ *   @title SanityRatesGasPrice contract
+ *   The contract is an update from the previous SanityRates contract, and provides
+ *   the following functionality:
+ *       - setting reasonable diff
+ *       - setting max gas price criteria for a trade
+ *       - setting sanity rates
+ *       - getting sanity rates
+ *
+ *       What's new from previous SanityRates contract:
+ *           1) Setting max gas price - essentially allows the reserve manager to
+ *              limit taker's trade gas price, in order to discourage front running
+ *              price updates.
+ *           2) Fix to https://github.com/KyberNetwork/smart-contracts/issues/168.
+ *           3) Update to Solidity 6.
+ */
+
+contract SanityRatesGasPrice is IKyberSanity, Withdrawable3, Utils5 {
+    struct SanityData {
+        uint168 tokenRate;
+        uint88 reasonableDiffInBps;
+    }
+
+    mapping(address => SanityData) public sanityData;
+    uint256 public maxGasPriceWei;
+
+    event SanityMaxGasPriceSet(uint256 maxGasPrice);
+
+    constructor(address _admin, uint256 _maxGasPriceWei) public Withdrawable3(_admin) {
+        setGasPrice(_maxGasPriceWei);
+    }
+
+    /// @dev set reasonableDiffInBps of a token to MAX_RATE if you want to ignore
```

Comment:
would write:
set reasonableDiffInBps of a token to MAX_RATE to avoid handling the price feed for this token.

Not clear what does it mean ignore sanity rate for it. (seems the sentence is in reserve level perspective...)

Reply:
Okay, changed it to this comment.